tests/aws/services/route53resolver/test_route53resolver.py

```diff
@@ -721,3 +722,221 @@ def test_list_firewall_domain_lists(self, cleanups, snapshot, aws_client):
 
         tag_result = aws_client.route53resolver.list_tags_for_resource(ResourceArn=arn)
         snapshot.match("list-tags-for-resource", tag_result)
+
+    @markers.aws.validated
+    @markers.snapshot.skip_snapshot_verify(paths=["$..Message"])
+    def test_list_firewall_rules_for_missing_rule_group(self, cleanups, snapshot, aws_client):
+        """Test listing firewall rules for a non-existing rule-group."""
+        missing_firewall_rule_group_id = f"missing-{short_uid()}"
+
+        snapshot.add_transformer(
+            [snapshot.transform.regex(missing_firewall_rule_group_id, "<firewall-rule-group-id>")]
+        )
```

Comment:
We can keep a static value of missing id here, say: `missing-id` and therefore can remove this additional transformer being added to this id.

Reply:
Thank you!
Done.